src/ucs/sys/topo/base/topo.c

```diff
@@ -367,23 +385,40 @@ ucs_topo_get_distance_sysfs(ucs_sys_device_t device1,
                             ucs_sys_device_t device2,
                             ucs_sys_dev_distance_t *distance)
 {
-    char path1[PATH_MAX], path2[PATH_MAX], common_path[PATH_MAX];
-    ucs_status_t status;
+    ucs_status_t status = UCS_OK;
+    char *path1;
```

Comment:
Single line?

Reply:
done

Reply:
Initialized them to NULL and aligned the lines.